config/crd/bases/o2ims.oran.openshift.io_clustertemplates.yaml

```diff
@@ -39,29 +39,61 @@ spec:
           spec:
             description: ClusterTemplateSpec defines the desired state of ClusterTemplate
             properties:
-              inputDataSchema:
-                description: InputDataSchema encapsulates all the schemas required
-                  for ClusterTemplate.
+              characteristics:
+                additionalProperties:
+                  type: string
+                description: Characteristics defines a List of key/value pairs describing
+                  characteristics associated with the template.>
+                type: object
+              description:
+                description: Description defines a Human readable description of the
+                  Template.
+                type: string
+              metadata:
+                additionalProperties:
+                  type: string
+                description: Metadata  defines a List of key/value pairs describing
+                  metadata associated with the template.>
+                type: object
+              name:
+                description: HumanReadableName defines a Human readable name of the
```

Comment:
Just wondering why you are using "HumanReadableName" instead of "Name" in this description?

Reply:
Fixed. I did not realized it translated to the schema. I did it initially to better distinguish it with metadata->name in the code